modules/ocf_syslog/files/ocf.conf

```diff
@@ -5,6 +5,16 @@ input(type="imtcp" port="514" ruleset="ocf-remote")
 module(load="imudp")
 input(type="imudp" port="514" ruleset="ocf-remote")
 
+# global options when writing to or creating files
+module(
+  load="builtin:omfile"
+  template="ocf-remote"
+  fileGroup="ocfstaff"
```

Comment:
Are you sure it is the best idea to make these logs readable by all ocfstaff? They do sometimes contain e.g. private keys that Puppet unwisely outputs and stuff like that...

Reply:
I'm just following the current log format, but I'd agree that sometimes puppet prints out things that we don't want shared by everyone. Maybe I can set the logs for hosts (not services) to be readable by `ocfroot` as the group? I'll give it a shot anyway

Reply:
Ok, I made logs from hosts only readable by `ocfroot` now in fc0fa59df72d91d2c349238291188b2513de5d28, and I think that's a good move since you're right that they sometimes contain more sensitive information.